mod-gobi/mod-gobi.postman_collection.json

```diff
@@ -397,6 +399,57 @@
 					},
 					"response": []
 				},
+				{
```

Comment:
Are you getting 404 on GET and POST `gobi/validate`?
![Screen Shot 2019-03-15 at 3 13 33 PM](https://user-images.githubusercontent.com/17807360/54456752-eefe6900-4735-11e9-8686-db5575f2045f.png)

Reply:
is the vagrant box testing-backend? What does /proxy/modules return? is mod-gobi installed on vagrant?

I have tested against folio-testing directly and it works
![image](https://user-images.githubusercontent.com/41596468/54457412-b95a7f80-4737-11e9-9563-146503b27239.png)

Reply:
Ok so for some reason [folio/testing ](https://app.vagrantup.com/folio/boxes/testing/versions/5.0.0-20190315.1784) does not have mod-gobi deployed. I was able to verify it in [folio/test-backend](https://app.vagrantup.com/folio/boxes/testing-backend/versions/5.0.0-20190315.1780) that it works fine.